scripts/regenerate_docs

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+docker pull pseudomuto/protoc-gen-doc
+
+docker run --rm \
+  -v $(pwd)/docs:/out \
+  -v $(pwd)/protos:/protos \
+  pseudomuto/protoc-gen-doc configs.proto --doc_opt=markdown,configs-reference.md
```

Comment:
let's see if we can generate lowerCameCase names

Reply:
There's no option for this currently - we would have to add these feature ourselves. I may do this!